pkg/cmd/pulumi/new.go

```diff
@@ -1083,6 +1090,21 @@ func templatesToOptionArrayAndMap(templates []workspace.Template,
 	return options, nameToTemplateMap
 }
 
+// formatDesc formats the template description nicely if the term width requires it to wrap around.
```

Comment:
This functions seems quite self-contained. 
Consider adding a unit test for it.